### ISSUE TITLE
[Bug]: Error in 3.5.1 

### ISSUE THREAD

**Issue by Dangry98** [mika-f/blender-drag-and-drop#30]
### What happened?

Addon is not supporteded in blender 3.5.1

Blender 3.5.1 had som important bug fixes, so if this addon coulde be updated it woulde be great! 

### Add-on Version

2.5.0

### Relevant log output

```Shell
[ERROR] exception: out of range
[ERROR] injector detected the unsupported version of Blender. Please upgrade Drag-and-Drop Support or downgrade Blender
[ERROR] detected version : 3.5.1
```

**Comment by mika-f** [comment]
This fix will be included in the next v2.6.0 release!

**Comment by Dangry98** [comment]
> This fix will be included in the next v2.6.0 release!

Awesome! Thank you so much for working on it! and the quick reply! :)

**Comment by Dangry98** [comment]
> This fix will be included in the next v2.6.0 release!

Wow! That was fast! Again thank you so much!
